README.md

```diff
@@ -1,70 +1,78 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Disney Tv Show
 
-## Available Scripts
+> A SPA app that shows the tv shows created by different Disney characters
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- React
+- Jest-test
+- Redux
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Netlify Live Demo Link]()
```

Comment:
- [ ] Kindly deploy your app using Heroku, Netlify or GitHub Pages and add the link to the deployed app here in the Readme.